camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -938,6 +939,22 @@ def pt2line(x0, y0, x1, y1, x2, y2):
         else:
             return None
 
+    def update_mask(self, img, screen):
+        """
+        Update or if None create a red mask with the same size of screen
+        and add on top of it detection areas displayed through green circles
+        """
+        height, width = img.shape[0], img.shape[1]
+        shape = screen.shape
+        scale_height, scale_width = height / shape[0], width / shape[1]
+        if self.mask is None:
+            self.mask = numpy.uint8(numpy.ones((shape[0], shape[1], 1)) * (0, 0, 255))
```

Comment:
Add a comment here that red mask is created here

Reply:
let's simplify this a bit: numpy.full((shape[0], shape[1], 3), (0, 0, 255), dtype=np.uint8)